src/app/home-page/home-page.component.scss

```diff
@@ -43,6 +42,18 @@ h2 {
     margin-bottom: 10px;
     padding-top: 30px;
   }
+
+  @media all and (min-width: 768px) {
+    max-width: 728px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  @media all and (min-width: 1152px) {
+    max-width: 1024px;
+  }
```

Comment:
Where are the styles for `@media all and (min-width: 1152px)`?

Reply:
@kilemensi For desktop we only need to update the max-width

Reply:
Added correction @kelvinkipruto https://github.com/CodeForAfrica/harassment-manager/pull/7#discussion_r993253808.

No, for desktop we need 1152 (small desktop) and 1440px, medium desktop. Check the other pages.